api/tests/Behat/bootstrap/v2/Registration/ActivateTrait.php

```diff
@@ -24,6 +24,14 @@ public function preRegistrationDetailsExistToAllowALayDeputyToRegisterForTheServ
         $this->existingPreRegistration = $this->fixtureHelper->createPreRegistration();
     }
 
+    /**
+     * @Given pre-registration details exist with no unicode characters$/
+     */
+    public function preRegistrationDetailsExistWithNoUnicodeCharacters()
+    {
+        $this->existingPreRegistration = $this->fixtureHelper->createPreRegistration('OPG102', 'PFA', 'O\'Shea');
```

Comment:
So we scaffold this as if it was propagated from Sirius. Do we know if there might be cases where Sirius has confused characters and so we need to normalise both sides. i.e. Sirius has `O’Shea `or `O`Shea` and the user enters `O'Shea`, opposite of the bug this fixes.

Reply:
Yeah so it can be the case that the data has been entered incorrectly on the Sirius side or even when the court order was served. 
I'm not sure if normalising the data we receive from Sirius is that good of an idea, as it's supposed to be our source of truth. I think in Lay cases where the names are wrong on Sirius, we usually wait for them to be updated there first and then allow the deputy to register. Although this is linked to another issue where we are not currently updating names when they get updated on Sirius.

It is a good point though, and maybe we need to make sure that both Digideps and Sirius are using the same character sets. 

I was also potentially thinking of using [the slugger component](https://symfony.com/doc/current/components/string.html#slugger) in Symfony, but was worried that it could convert more characters than expected and result in some of the problems you mentioned above. So I stuck to the known characters that were causing issues for now.

Reply:
Yeah, I wouldn't blanket change the way we ingest, but at compare time it would be useful to be comparing like and like. Slugger would likely convert too much. 

It's not even the character sets, so much as confusable characters and variance on who input data. You used to get the same issue with content pasted from word, where everything became smart quotes not apostrophes. The list is quite large: https://util.unicode.org/UnicodeJsps/confusables.jsp?a=%27&r=None